mypy/options.py

```diff
@@ -129,7 +129,7 @@ def __init__(self) -> None:
         self.scripts_are_modules = False
 
         # Config file name
-        self.config_file = None  # type: Optional[str]
+        self.config_file = 'setup.cfg'  # type: Optional[str]
```

Comment:
Isn't this just `str` now? (no longer `Optional`?)

Reply:
I think it isn't. Looks like it's possible to have None if there's no config file; but when running the test, there is. I'm not sure about that though; I wonder why ArgSuite doesn't fail on master.

Reply:
What's the reason for this change?

Reply:
This change doesn't seem to be related to the rest of PR. Can you remove this?

Reply:
It is needed because (as far as I understand) pytest uses the config file, so the default option is not None. I'm not sure this change is the right one; another option I can think of is changing this value inside the test.

Reply:
But this is the mypy config file, which I don't expect to be directly relevant for pytest. Pytest config file should be configured somewhere else (not sure how this works exactly, though) -- and pytest already has a config file (`pytest.ini`) which seems to be used. Does this PR do something that requires changes to this? Or maybe I'm talking about a different config file?

Reply:
I think I might have confused something in my previous response (it's been a while since I tried to figure this out), but note that `SHARED_CONFIG_FILES = ('setup.cfg',)` is hardcoded in `main.py`. Somewhere in `parse_config_file()` there's probably a difference between how myunit and pytest work, so one the former uses `None` and the latter uses `'setup.cfg'`.

Reply:
Running myunit, `os.path.exists('setup.cfg')` is false. Running pytest, it is true.

Reply:
I'm trying a different approach - simply create a new directory and chdir into it.

Reply:
Failed - changing directories did not work on Windows.